src/main/java/pl/craftserve/radiation/LugolsIodinePotion.java

```diff
@@ -185,20 +190,23 @@ public void onBrew(BrewEvent event) {
 
     private void convert(PotionMeta potionMeta) {
         Objects.requireNonNull(potionMeta, "potionMeta");
+        int durationMinutes = (int) this.config.duration().toMinutes();
 
-        potionMeta.setDisplayName(ChatColor.AQUA + this.name);
-        potionMeta.setLore(Collections.singletonList(ChatColor.BLUE + "Odporność na promieniowanie (" + this.duration + ":00)"));
+        potionMeta.setDisplayName(ChatColor.AQUA + this.config.name());
+        potionMeta.setLore(Collections.singletonList(ChatColor.BLUE + "Odporność na promieniowanie (" + durationMinutes + ":00)"));
```

Comment:
Jak juz to edytujemy to zdecydowalbym sie na uzycie jednego okreslenia wszedzie, tj. radiacja lub promieniowanie

Reply:
To też wrzucę do configu. Obecnie jest to jedyny strng zapisany w kodzie.